### DISCUSSION TITLE
Managed Kubernetes

### DISCUSSION THREAD

**Discussion by oleg-glagovskiy** [ubicloud/ubicloud#1470] [Ideas]
Did you have idea of have also managed kubernetes? If yes, how far is it in your plans?
Currently I guess there are two main downsides of Hetzner vs AWS or whatever - managed stuff. And most used managed stuff is k8s (as well as managed DBs).

**Comment by byucesoy** [comment]
Hi @oleg-glagovskiy,
K8s is one of the most requested managed services. Currently, we don't have a timeline for it, but it is high in our priority list.

Anyone who is interested with K8s, feel free to upvote the Oleg's message to show the interest.
